presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -179,6 +180,11 @@ public SystemSessionProperties(
                         "Number of partitions for distributed joins and aggregations",
                         queryManagerConfig.getHashPartitionCount(),
                         false),
+                stringProperty(
+                        PARTITIONING_PROVIDER_CATALOG,
+                        "Partitioning provider catalog",
```

Comment:
Looks like we need a better description here as this repeats the property name. Maybe something like `Name of the catalog providing custom partitioning.` or sth along those lines.